docs/demo/gaiden-css/components/list/app/demo.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+
+  <style>
+    body {
+      background-color: #f9f9f9;
+    }
+  </style>
+</head>
+
+<body>
+  <article class="col-small-6 col-medium-4 col-normal-3 col-large-2" style="display: block">
+    <header>
+      <h5 class="list__title">Topic List Header</h5>
```

Comment:
@felipeorlando this is strange because there is no `.list` element to have` .list__header`.